CHANGELOG.md

```diff
@@ -5,6 +5,12 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [4.0.0] Unreleased
```

Comment:
Since this PR introduces potentially breaking changes (again AFAIK, no one is using the currency component), I assume the next version would be `4.0.0`?

Reply:
Yea that’s right